dask/sizeof.py

```diff
@@ -101,3 +101,27 @@ def sizeof_spmatrix_dok(s):
     @sizeof.register(sparse.spmatrix)
     def sizeof_spmatrix(s):
         return sum(sizeof(v) for v in s.__dict__.values())
+
+
+@sizeof.register_lazy("pyarrow")
+def register_pyarrow():
+    import pyarrow as pa
+
+    def _get_col_size(col):
```

Comment:
Question / style nitpick: can you put this implementation in the `sizeof_pyarrow_column` and just call `sizeof(col)` in the for loop for Table below?

```python
for col in table.itercolumns():
    p += sizeof(col)
```

I think that the registration happens first, so things should work out.

Reply:
Right - I only avoided this, because sizeof_pyarrow_column is adding the 1000-byte serialization overhead for each column.

Reply:
Ahh, that make sense.